src/agent/test/e2e/logs-count.e2e-spec.ts

```diff
@@ -0,0 +1,36 @@
+// Import Node.js Dependencies
+import { describe, it } from "node:test";
+import assert from "node:assert";
+import path from "node:path";
+import { once } from "node:events";
+
+// Import Internal Dependencies
+import { TestingNotifier } from "../FT/mocks/sigyn-test-notifier";
+import { setupEnvironment } from "./setup";
+import { start } from "../../src";
+
+// CONSTANTS
+const kConfigLocation = path.join(__dirname, "./configs/count-logs");
+const kTestingNotifier = TestingNotifier.getInstance();
+
+describe("Given a rule with 'on.count: 1'", () => {
+  it("Should send an alert when at least one log is found ", async() => {
+    assert.equal(kTestingNotifier.notifCount, 0);
+
+    const { fetchLoggingEndpoint, server } = await setupEnvironment({ grafanaHost: "https://localhost:3000" });
+    await fetchLoggingEndpoint();
+
+    const scheduler = await start(kConfigLocation);
+    await once(kTestingNotifier, "notif");
```

Comment:
Add an AbortSignal.timeout here just to be sure the event is resolve before X ms ?